axelrod/tests/unit/test_classification.py

```diff
@@ -1,11 +1,106 @@
 """Tests for the classification."""
 
+import os
 import unittest
+from typing import Any, Text
+import warnings
+import yaml
 
 import axelrod as axl
+from axelrod.classifier import (
+    Classifier,
+    Classifiers,
+    _Classifiers,
+    memory_depth,
+    rebuild_classifier_table,
+)
+from axelrod.player import Player
+
+
+class TitForTatWithEmptyClassifier(Player):
+    """
+    Same name as TitForTat, but with empty classifier.
+    """
+
+    # Classifiers are looked up by name, so only the name matters.
+    name = "Tit For Tat"
+    classifier = {}
+
+
+class TitForTatWithNonTrivialInitialzer(Player):
+    """
+    Same name as TitForTat, but with empty classifier.
+    """
+
+    def __init__(self, x: Any):
+        pass
```

Comment:
1 suggestion would be

```suggestion
        pass  # pragma: no cover
```

Instead of doing something hacky we just say we don't need this line to be checked.